pkg/engagement/usecases/usecases.go

```diff
@@ -2,23 +2,30 @@ package usecases
 
 import (
 	"github.com/savannahghi/engagement/pkg/engagement/infrastructure"
+	"github.com/savannahghi/engagement/pkg/engagement/usecases/fcm"
 	"github.com/savannahghi/engagement/pkg/engagement/usecases/feed"
 )
 
 // Interactor is an implementation of the usecases interface
 type Interactor struct {
 	*feed.UseCaseImpl
 	*feed.NotificationImpl
+	*fcm.PushImpl
+	*fcm.FCMImpl
 }
 
 // NewInteractor initializes a new usecases interactor
 func NewInteractor(infrastructure infrastructure.Interactor) Interactor {
```

Comment:
This was a bit of of an oversight from my end. Rename this to `NewUsecasesInteractor`